### PR TITLE
[T4A3][T13-B1] Nathan Ohun Kwon 

### PR DESCRIPTION
**Pull request by kwonn**
Ready for review.

### CODE REVIEW

File: src/seedu/addressbook/data/person/Address.java [hunk 1]
```diff
@@ -6,26 +6,22 @@
...
-    private boolean isPrivate;
-
     /**
      * Validates given address.
      *
      * @throws IllegalValueException if given address string is invalid.
      */
     public Address(String address, boolean isPrivate) throws IllegalValueException {
-        this.isPrivate = isPrivate;
+        super(address, isPrivate);
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Address.java:21]: think how you can push the validation logic as well to the parent class.

File: src/seedu/addressbook/data/person/Contact.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Peron's contact in the address book
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Contact.java:4]: :+1:

File: src/seedu/addressbook/data/person/Contact.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+ */
+
+public class Contact {
+
+    public final String value;
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Contact.java:9]: try to make it private. in general, start off with private members, then use protected or public only if absolutely necessary.

**Comment by okkhoy** [comment]
@kwonn 
Some comments added. Please ack & close the PR after reading comments.

**Comment by kwonn** [comment]
Understood. Thank you for the comments!

**Pull request closed without merging**